homeassistant/components/scene/__init__.py

```diff
@@ -32,15 +32,13 @@ def _hass_domain_validator(config: dict[str, Any]) -> dict[str, Any]:
 
 def _platform_validator(config: dict[str, Any]) -> dict[str, Any]:
     """Validate it is a valid  platform."""
+    platform_name = config[CONF_PLATFORM]
     try:
-        platform = importlib.import_module(f".{config[CONF_PLATFORM]}", __name__)
```

Comment:
AFAICT these no longer exist

Reply:
Theoretically a custom integration could overwrite the scene integration and put a platform there. Sounds a bit unlikely though.

Reply:
It wouldn't use this code then anyway, so I don't think there's a case where we actually need this code.